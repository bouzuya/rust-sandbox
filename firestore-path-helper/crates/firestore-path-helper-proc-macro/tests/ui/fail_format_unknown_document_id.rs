fn main() {
    firestore_path_helper_proc_macro::firestore_path_helper!(
        "users/{user_id}/repos/{repo_id}",
        user_id = String,
        // Missing repo_id argument
    );
}
