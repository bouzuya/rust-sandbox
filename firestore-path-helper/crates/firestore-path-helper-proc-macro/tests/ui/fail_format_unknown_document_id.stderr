error: document id segment 'repo_id' does not match any argument
 --> tests/ui/fail_format_unknown_document_id.rs:3:9
  |
3 |         "users/{user_id}/repos/{repo_id}",
  |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
